flow/Trace.cpp

```diff
@@ -576,16 +564,41 @@ TraceEventFields LatestEventCache::getLatestError() {
 
 static TraceLog g_traceLog;
 
-bool selectTraceFormatter(std::string format) {
-	ASSERT(!g_traceLog.isOpen());
+namespace {
```

Comment:
I also restructured the code so that we only enumerate the list of recognized formatters once.